### PR TITLE
[12.0][MIG] stock_barcodes_automatic_entry

### PR DESCRIPTION
**Pull request by AaronHForgeFlow**
Standard migration

### CODE REVIEW

File: stock_barcodes_automatic_entry/__manifest__.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
         with the class 'barcode-automatic-entry' after a barcode scanned has 
         been processed.
     """,
-    'version': '11.0.1.0.0',
+    'version': '12.0.1.0.0',
     'license': 'AGPL-3',
     "maintainers": ['ageficent'],
     'author': 'Eficent Business and IT Consulting Services S.L.,'
```
Comment (AdriaGForgeFlow) [inline stock_barcodes_automatic_entry/__manifest__.py]: Can you update this and the copyrights as well?

**Review by eLBati — APPROVED** [review]
:+1: After https://github.com/OCA/stock-logistics-barcode/pull/267#discussion_r404778720

**Comment by eLBati** [comment]
@AaronHForgeFlow please squash the last 2 commits

**Comment by AaronHForgeFlow** [comment]
@eLBati done! and green  :)

**Review by AdriaGForgeFlow — APPROVED** [review]
Code review, no major changes :+1:

**Comment by rousseldenis** [comment]
@AaronHForgeFlow Do you want to squash last commit or let as is ?

**Comment by AaronHForgeFlow** [comment]
For me it is ok like this, it is more clear.

**Review by marcelsavegnago — APPROVED** [review]

**Comment by eLBati** [comment]
Is this mergeable?

**Comment by AaronHForgeFlow** [comment]
I think so, it is used in production environments.

**Comment by eLBati** [comment]
@OCA/logistics-maintainers merge?

**Comment by rousseldenis** [comment]
@AaronHForgeFlow Ok, let's merge it as it was done like that but:

- Don't use plural in module name.
- IMHO, this is a transversal module as the function can be used in any model. It should be stored in server-tools or web and be renamed.

**Pull request merged**